.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules/
 package-lock.json
+yarn.lock
```

Comment:
Why should we remove the yarn lock file? It pins all our dependecies, I would prefer keeping it

Reply:
[`yarn.lock` only pins dependencies for applications, not libraries.](https://yarnpkg.com/en/docs/yarn-lock#toc-current-package-only)

What that means is that if somebody adds `wasm-pack-plugin` to their `dependencies` then the `yarn.lock` is completely ignored. Or in other words, `yarn.lock` is ignored for all of the *users* of `wasm-pack-plugin`.

The only time when `yarn.lock` will have an effect is when somebody is *developing* `wasm-pack-plugin`. But that's bad, since it means the user is getting the latest versions but the developer isn't, so this can create bugs.

The same is true for `package-lock.json`. This was an intentional design choice done to avoid dependency hell.

Reply:
I see, I'm fine with the change. Thanks for the clarification.